player/command.c

```diff
@@ -2381,6 +2381,34 @@ static int panscan_property_helper(void *ctx, struct m_property *prop,
     return r;
 }
 
+// Properties retrieved through VOCTRL_PERFORMANCE_DATA
+static int perfdata_property_helper(void *ctx, struct m_property *prop,
+                                    int action, void *arg)
+{
+    MPContext *mpctx = ctx;
+    if (!mpctx->video_out)
+        return M_PROPERTY_UNAVAILABLE;
+
+    struct voctrl_performance_data data = {0};
+    if (vo_control(mpctx->video_out, VOCTRL_PERFORMANCE_DATA, &data) <= 0)
+        return M_PROPERTY_UNAVAILABLE;
```

Comment:
Would it be a better design to have a global struct inside MPContext or something that the VO updates regularly, with the property code only reading these fields?